src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.CSharp.targets

```diff
@@ -12,6 +12,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <AppDesignerFolder Condition="'$(AppDesignerFolder)' == ''">Properties</AppDesignerFolder>
```

Comment:
I am against it, but I don't have a better workaround. Basically, the AppDesigner folder is just a normal folder and it's contents should be considered the same as the rest of the tree. The problem is that we put the debugger settings under there - and they don't want to publish that file. In the past we would use different item types such as `<None/>` to represent that concept, but we're publishing None, right @dsplaisted?